fairyfishgui.py

```diff
@@ -298,11 +320,29 @@ def format_info(info):
                 pass
         self.engine_thread = threading.Thread(target=read_output, daemon=True)
         self.engine_thread.start()
+        self.engine.initialize()
         self.engine.setoption('UCI_Variant', self.board.state.variant)
         self.engine.newgame()
         self.engine.position(self.board.state.start_fen, self.board.state.moves)
         self.engine.analyze()
 
+    def set_engine_option(self, nnue=None, threads=None):
+        if self.engine and not self.engine.paused and (nnue or threads):
+            self.engine.stop()
+            self.engine.paused = False
+        if nnue:
+            if self.engine:
+                self.engine.setoption('EvalFile',nnue)
+            else:
+                self.engine_settings['EvalFiles'] = nnue
```

Comment:
Thanks for your updates. I think there is just one typo here, otherwise looks good to me.